### ISSUE TITLE
Remove the marked: API from WebView queries.

### ISSUE THREAD

**Issue by jmoody** [calabash/calabash-ios-server#247]
* **Cannot query UIWebView by accessibilityIdentifier or accessibilityLabel** [#735](https://github.com/calabash/calabash-ios/issues/735)

This is a breaking change and might need to wait for Calabash 2.0.
